app/src/main/res/values/strings.xml

```diff
@@ -656,8 +656,8 @@ How to translate with transifex:
     <string name="nc_dialog_outdated_client_description">The app is too old and no longer supported by this server. Please update.</string>
     <string name="nc_dialog_outdated_client_option_update">Update</string>
     <string name="nc_switch_account">Switch account</string>
-    <string name="nc_dialog_maintenance_mode">Maintenance mode</string>
-    <string name="nc_dialog_maintenance_mode_description">Server is currently in maintenance mode.</string>
+    <string name="nc_dialog_maintenance_mode_description">Server is currently in maintenance mode. Please try again
+        later.</string>
```

Comment:
could use a different string name, so it doesn't conflict the translation sync until we backported